src/System.Runtime/tests/System.Runtime.Tests.targets

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+    <Target Name="CopyFilesToNonTPADir" BeforeTargets="RunTestsForProject" >
```

Comment:
I chatted with Rahul a bit and I think this approach might work, but looking at this it's unnecessary.  It could work (would have to experimentally verify) because as long as it's in the bin dir before archiving happens, we'll archive things as-is. 

However, @rahku  please just check out the DestinationDir property of SupplementalTestData: https://github.com/dotnet/buildtools/blob/01ef134d9c7fc08f63a5e5d2fc06e3ecc69f5386/src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets#L158  

Setting this metadata value on that file will use the logic in the link above to either 1) Copy it to the subfolder it gets binplaced to... this is for the case of things like resources that need to be in subfolders or 2) Copy it wherever you want in a subfolder specified by DestinationDir.

Between these two behaviors, you should be able to get most any combination of bin-placing where you like without adding a custom target.